library/src/interactor_impl.cxx

```diff
@@ -796,21 +782,75 @@ bool interactor_impl::triggerCommand(std::string_view command)
   return false;
 }
 
+//----------------------------------------------------------------------------
+interactor& interactor_impl::createDefaultInteractionsCommands()
```

Comment:
```suggestion
interactor& interactor_impl::initializeDefaultInteractionsCommands()
```
maybe `initialize` makes it more explicit that it will set/reset all the interactions to the defaults ones (by clearing first) rather that add them?

Reply:
agree